compiler/src/main/kotlin/software/amazon/lastmile/kotlin/inject/anvil/processor/MergeComponentProcessor.kt

```diff
@@ -173,9 +190,12 @@ internal class MergeComponentProcessor(
         resolver: Resolver,
         contributingAnnotations: Sequence<String>,
     ): Boolean {
-        return contributingAnnotations.flatMap {
-            resolver.getNewSymbolsWithAnnotation(it)
-        }.any { true }
+        return contributingAnnotations
+            .distinct() // Save duplicate lookups
```

Comment:
This should be moved to line 122.

Reply:
[83df1a3](https://github.com/amzn/kotlin-inject-anvil/pull/25/commits/83df1a3fd450d33328a59eb1eb4b4436599e706c)